### DISCUSSION TITLE
Feign wants to join Commonhaus!

### DISCUSSION THREAD

**Discussion by velo** [commonhaus/foundation#155] [Joining Commonhaus]
### Project information

- **Project name**: Feign
- **Project website**: https://github.com/OpenFeign/feign
- **Code repository**: https://github.com/OpenFeign/feign
- **License**: Apache License V2

### Do you have authority to represent this project?

- [X] I own or maintain this project

### Describe your project

Feign is a Java to HTTP client binder inspired by [Retrofit](https://github.com/square/retrofit), [JAXRS-2.0](https://jax-rs-spec.java.net/nonav/2.0/apidocs/index.html), and [WebSocket](http://www.oracle.com/technetwork/articles/java/jsr356-1937161.html). Feign's first goal was reducing the complexity of binding [Denominator](https://github.com/Netflix/Denominator) uniformly to HTTP APIs regardless of [ReSTfulness](http://www.slideshare.net/adrianfcole/99problems).

### Things that must be done (bootstrap phase)

- [x] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval (with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [x] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [x] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by kenfinnigan** [comment]
@velo please open a PR with the above outlined information to commence the process of joining

**Comment by ebullient** [comment]
Resolved by #165 

Welcome to Commonhaus! 🎉 💖 🚀

**Reply by ebullient** [reply]
@velo 

I need you to accept the invitation to the Commonhaus organization so we can walk you through the next steps. ;)

**Reply by velo** [reply]
I got no invitation
![Screenshot_20240615-092025](https://github.com/commonhaus/foundation/assets/136590/6c729681-fd73-46b4-b3ec-93f1f5bb6b32)

**Reply by ebullient** [reply]
This will be a GitHub notification, see https://github.com/settings/organizations

**Reply by velo** [reply]
Ow, I would never found that by myself :)

Thanks and done
